packages/edit-site/src/hooks/commands/use-edit-mode-commands.js

```diff
@@ -94,12 +100,20 @@ function useManipulateDocumentCommands() {
 	if ( isTemplateRevertable( template ) && ! hasPageContentFocus ) {
 		const label =
 			template.type === 'wp_template'
-				? __( 'Reset template' )
-				: __( 'Reset template part' );
+				? /* translators: %1$s: template title */
+				  sprintf(
+						'Reset template — %s',
+						decodeEntities( template.title )
+				  )
+				: /* translators: %1$s: template part title */
+				  sprintf(
+						'Reset template part — %s',
+						decodeEntities( template.title )
+				  );
 		commands.push( {
 			name: 'core/reset-template',
 			label,
-			icon: backup,
+			icon: rotateLeft,
```

Comment:
How about considering the RTL language?

```javascript
isRTL() ? rotateRight : rotateLeft
```

I also think this icon literally means "rotate". I am wondering if there is any other appropriate icon.

![rotate](https://github.com/WordPress/gutenberg/assets/54422211/b3291230-37df-48d5-b044-ed692c585382)